README.md

```diff
@@ -162,6 +162,8 @@ kedro viz --save-file=filename.json
 
 We also recommend wrapping the `Kedro-Viz` component with a parent HTML/JSX element that has a specified height (as seen in the above example) in order for Kedro-Viz to be styled properly.
 
+**_For more examples on how to visualise with kedro-viz, please refer to the [Documentation](https://docs.kedro.org/en/stable/visualisation/)_**
```

Comment:
```suggestion
**_The documentation contains [additional examples on how to visualise with kedro-viz](https://docs.kedro.org/en/stable/visualisation/)_**
```

Reply:
hey yes of course, thanks for the suggestion